package/boot/uboot-mediatek/Makefile

```diff
@@ -236,6 +236,18 @@ define U-Boot/mt7986_bananapi_bpi-r3-nor
   FIP_COMPRESS:=1
 endef
 
+define U-Boot/mt7986_xiaomi_redmi-router-ax6000-snand
```

Comment:
The AX6000 *always* boots from SNAND. Hence there is no need to carry the word `snand` in the name.

Reply:
Done. Thanks